torch/distributions/constraints.py

```diff
@@ -545,6 +545,7 @@ def check(self, value):
 real = _Real()
 real_vector = independent(real, 1)
 positive = _GreaterThan(0.)
+nonnegative = _GreaterThanEq(0.)
```

Comment:
Would you mind also registering this like `positive` in the constraint registry, so as to avoid breaking downstream code that uses the constraint registry to register constrained parameters, e.g. when learning parameters of Poisson distributions?
```diff
diff --git a/torch/distributions/constraint_registry.py b/torch/distributions/constraint_registry.py
index cbe987e72c..c03f0ad02d 100644
--- a/torch/distributions/constraint_registry.py
+++ b/torch/distributions/constraint_registry.py
@@ -173,7 +173,9 @@ def _transform_to_independent(constraint):


 @biject_to.register(constraints.positive)
+@biject_to.register(constraints.nonnegative)
 @transform_to.register(constraints.positive)
+@transform_to.register(constraints.nonnegative)
 def _transform_to_positive(constraint):
     return transforms.ExpTransform()
```